32blit-pico/led.cpp

```diff
@@ -31,6 +49,16 @@ void init_led() {
   bi_decl(bi_1pin_with_name(led_pins[0], "Red LED"));
   bi_decl(bi_1pin_with_name(led_pins[1], "Green LED"));
   bi_decl(bi_1pin_with_name(led_pins[2], "Blue LED"));
+#elif defined(LED_WS2812_PIN)
+  PIO pio = pio0;
+  uint pio_offset = pio_add_program(pio, &ws2812_program);
+  pio_sm = pio_claim_unused_sm(pio, true);
```

Comment:
`required` is `true` here, so the error check is redundant. (It'll panic before it gets there)

Reply:
Also, probably want a `bi_decl` for the pin.

Reply:
> `required` is `true` here, so the error check is redundant. (It'll panic before it gets there)

My bad ! :)

Reply:
> Also, probably want a `bi_decl` for the pin.

I didn't know about bi_decl, and a very little of picotool. This is very nice !
```
File cmake-build-debug-pico/examples/saves/saves.uf2:

Program Information
 name:            saves
 version:         v1.0.0
 web site:        https://github.com/32blit/32blit-sdk
 description:     Save data example.
 features:        UART stdin / stdout
 binary start:    0x10000000
 binary end:      0x10018460
 embedded drive:  0x10600000-0x10800000 (2048K): 32Blit

Fixed Pin Information
 0:   UART0 TX, X Button
 1:   UART0 RX, Y Button
 2:   DOWN Button
 3:   LEFT Button
 7:   UP Button
 8:   Display D/C
 9:   Display CS
 10:  Display SCK
 11:  Display TX
 12:  Display Backlight
 13:  Display Reset
 16:  NeoPixel RGB LED (WS2812)
 24:  B Button
 25:  A Button
 29:  RIGHT Button

Build Information
 sdk version:       1.5.1
 pico_board:        adafruit_feather_rp2040
 boot2_name:        boot2_generic_03h
 build date:        Jan  5 2024
 build attributes:  Debug
```

Reply:
Fixed in d2d5f7f28b8c6d0a3afe1a032bbb06d97f1f02d4 i hope.